### PR TITLE
Module 1 Awesome Books Library styling

### PR DESCRIPTION
**Pull request by the-catalystmc**
In this project we:

- Make sure that there are no linter errors.
- Make sure that we use the correct GitHub flow.
- Add CSS styles to the application to match the given wireframe.
- Concentrate on the general layout, rather than the details.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,23 +9,26 @@
 </head>
 <body>
     <main>
-        <section class="container">
-            <h2 class="books__header--title">
-                Awesome Books
-            </h2>
+        <h2 class="books__header--title">
+            All awesome books
+        </h2>
```
Comment (voscarmv) [inline index.html]: - [x] Please use proper semantic HTML here by using the `<header>` tag for the document title, and the `<main>` tag for the rest of the document.

File: index.html [hunk 2]
```diff
@@ -9,23 +9,26 @@
...
         <form class="add">
-            <input type="text" class="add__title" id="add__title" name="book_title" placeholder="Title" required>
-            <input type="text" class="add__author" id="add__author" name="book_author" placeholder="Author" required>
-            <input type="submit" class="add__add-btn" value="Add">
+            <input type="text" class="add__title input" id="add__title" name="book_title" placeholder="Title" required>
+            <input type="text" class="add__author input" id="add__author" name="book_author" placeholder="Author" required>
+            <input type="submit" class="add__add-btn input" value="Add">
         </form>
     </main>
     <script src="app.js"></script>
```
Comment (voscarmv) [inline index.html]: - [x] Please user proper 2-space indentation for your HTML in order to keep with the best practices.

**Review by voscarmv — CHANGES_REQUESTED** [review]
Hi @the-catalystmc , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- I've left some comments regarding best HTML practices. Please read them and fix the issues mentioned there!

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Fixing the broken Live Demo link on your README file.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**